lib/crewai-tools/src/crewai_tools/tools/skim_reader_tool/README.md

```
@@ -0,0 +1,156 @@
+print(crew.kickoff())
+```
+
+The agent decides when to call the Skim Web Reader, Skim returns clean Markdown (~4x smaller than raw HTML), and the model gets a faster, cheaper read.
+
+---
+
+## Output shape
+
+`SkimReaderTool` returns Markdown with a YAML frontmatter block of the page metadata:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)